WalletWasabi/JsonConverters/EndPointJsonConverter.cs

```diff
@@ -44,7 +38,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 		}
 
 		/// <inheritdoc />
-		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+		public override void WriteJson(JsonWriter writer, EndPoint? value, JsonSerializer serializer)
```

Comment:
Please look 2 lines below. It is unnecessary to check if the type is `EndPoint` because it cannot be anything else (except `null` ofc).